node/integrationtest/yarn.js

```diff
@@ -102180,7 +102180,7 @@ function allocUnsafe (size) {
   if (Buffer.allocUnsafe) {
     return Buffer.allocUnsafe(size)
   } else {
-    return new Buffer(size)
+    return Buffer.from(size)
```

Comment:
This also should be Buffer.alloc?

Reply:
Yep! Fixed.

Reply:
I wonder if this should be left as it was. It looks like this was a fallback for the case where `Buffer.allocUnsafe()` did not exist. `Buffer.allocUnsafe()` and `Buffer.alloc()` were added in the same version of Node, so this might not be a good fallback. That said, those methods were added in Node 5, which has been end-of-life for years now, so it probably doesn't matter. 🤷

Reply:
I agree. Let's just match Node for now.